website/blog/2022-12-21-databend-weekly-73.md

```diff
@@ -0,0 +1,148 @@
+**Multiple Catalogs**
+
+- implement show tables (from|in catalog.database) ([#9153](https://github.com/datafuselabs/databend/pull/9153))
+
+**Planner**
+
+- introduce histogram in column statistics ([#9310](https://github.com/datafuselabs/databend/pull/9310))
+
+**Query**
+
+- support attaching stage for insert values ([#9249](https://github.com/datafuselabs/databend/pull/9249))
+- add native format in fuse table ([#9279](https://github.com/datafuselabs/databend/pull/9279))
+- add internal_enable_sandbox_tenant config and sandbox_tenant ([#9277](https://github.com/datafuselabs/databend/pull/9277))
+
+**Sqllogictest**
+
+- introduce rust native sqllogictest framework ([#9150](https://github.com/datafuselabs/databend/pull/9150))
+
+### Code Refactoring :tada:
+
+**\***
+
+- unify apply_file_format_options for copy & insert ([#9323](https://github.com/datafuselabs/databend/pull/9323))
+
+**IO**
+
+- remove unused code ([#9266](https://github.com/datafuselabs/databend/pull/9266))
+
+**meta**
+
+- test watcher count ([#9324](https://github.com/datafuselabs/databend/pull/9324))
+
+**Planner**
+
+- replace TableContext in planner with PlannerContext ([#9290](https://github.com/datafuselabs/databend/pull/9290))
+
+### Bug Fixes :wrench:
+
+**Base**
+
+- try fix SIGABRT when catch unwind ([#9269](https://github.com/datafuselabs/databend/pull/9269))
+- replace #[thread_local] to thread_local macro ([#9280](https://github.com/datafuselabs/databend/pull/9280))
+
+**Query**
+
+- fix unknown database in query without relation to this database ([#9250](https://github.com/datafuselabs/databend/pull/9250))
+- fix wrong current_role when drop the role ([#9276](https://github.com/datafuselabs/databend/pull/9276))
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+#### Introduce Rust Native Sqllogictest Framework
+
+Sqllogictest is a program designed to verify that an SQL database engine computes correct results by comparing the results to identical queries from other SQL database engines. 
+
+In the past, Databend has run this test using a program written in Python and has migrated a large number of test cases from other successful databases. Recently, we have reimplemented this program with [sqllogictest-rs](https://github.com/risinglightdb/sqllogictest-rs) using Rust.
+
+**Learn More**
+
+- [Doc | sqllogictest](https://github.com/datafuselabs/databend/tree/main/tests/sqllogictests)
+- [PR | introduce rust native sqllogictest framework](https://github.com/datafuselabs/databend/pull/9150)
+
+#### Experimental: Native Format
+
+[PA](https://github.com/sundy-li/pa) is a native storage format based on Apache Arrow. PA is similar to [Arrow IPC](https://arrow.apache.org/docs/python/ipc.html) and is primarily aimed at optimizing the storage layer. We hope to use it in databend as another storage_format, and it is currently in a very early stage.
+
+We tried to introduce it as a native storage format and got a considerable performance.
```

Comment:
Databend is introducing PA as a native storage format in the hope of getting a performance boost, though it's still at an early stage of development.